Common/FrameworkBundle.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  FrameworkBundle.swift
+//  LoopKit
+//
+//  Created by Nathaniel Hamming on 2020-07-15.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+class FrameworkBundle {
```

Comment:
@ps2 had expressed concerns over propagating this code strategy in one of my prior PRs.  I'm not blocking this PR for that, but rather raising an awareness.

Reply:
I don't have a preference for this code, just like the reduced redundancy. Curious what the concern was.